app/src/App/App.tsx

```diff
@@ -313,6 +323,22 @@ const App: React.FC = () => {
               {...inventoryServiceProps}
               component={ConnectedServicePointList}
             />
+            <PrivateComponent
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+              exact
+              {...BaseProps}
+              path={INVENTORY_ADD_SERVICE_POINT}
```

Comment:
In future implementations could we pass the URL as a prop from the cient instead of package to client?